packages/edit-site/src/components/dataviews/field-types.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import Media from '../../components/media';
+
+export const coreFieldTypes = {
+	string: {
+		view:
```

Comment:
I saw that we changed from `cell` to `render` to make rendering tanstack independent. Do we need to change from `render` to `view`?

I don't know when/why we decided on `render`, but I find it confusing that I just learned about it, and it's changing again 🙈 If I had to chose, I'd still keep `render`. `view` is already taken for other things (view types: table, grid, etc.).